ansible/lifecycle_ec2.yml

```diff
@@ -11,6 +11,7 @@
         wait: no
         filters:
           "tag:guid": "{{ guid }}"
+          instance-state-name: running
```

Comment:
Out of curiousity, why are these needed. If you tell the module to stop the instance...it knows to stop it if it is running and not stop it if if it stopped.

Reply:
It's a safeguard when this is run shortly after provisioning. Because then the terminated bootstrap VM is still there - and the module will throw up...